versions.props

```diff
@@ -8,7 +8,7 @@ com.fasterxml.woodstox:woodstox-core=6.5.0
 com.github.ben-manes.caffeine:caffeine=3.1.4
 com.github.spotbugs:*=4.7.3
 com.github.stephenc.jcip:jcip-annotations=1.0-1
-com.google.cloud:google-cloud-bom=0.184.0
```

Comment:
`io.grpc:grpc-*=1.51.0` needs to be changed to `io.grpc:grpc-*=1.53.0`

Reply:
This will cover https://github.com/apache/solr/pull/1418

Reply:
Feel free to complete this...

Reply:
fixed in 153fcbafce43823d57ad1ef0ccbad4dfe3861735